cmd/mongocli/mongocli.go

```diff
@@ -38,6 +40,59 @@ func Execute(ctx context.Context) {
 	}
 }
 
+func updateMongoCLIConfigPath() {
+	mongoCLIConfigHome, err := config.MongoCLIConfigHome()
+	if err != nil {
+		return
+	}
+
+	mongoCLIConfigPath := fmt.Sprintf("%s/%s", mongoCLIConfigHome, "config.toml")
+	f, err := os.Open(mongoCLIConfigPath) // if config.toml is already there, exit
+	if err == nil {
+		return
+	}
+	defer f.Close()
+
+	oldMongoCLIConfigHome, err := config.OldMongoCLIConfigHome()
+	if err != nil {
+		return
+	}
+
+	oldMongoCLIConfigPath := fmt.Sprintf("%s/%s", oldMongoCLIConfigHome, "mongocli.toml")
+	in, err := os.Open(oldMongoCLIConfigPath)
+	if err != nil {
+		return
+	}
+	defer in.Close()
+
+	_, _ = fmt.Fprintf(os.Stderr, `MongoCLI uses a new config path. Copying mongocli.toml content to: %s
+`, mongoCLIConfigPath)
+
+	_, err = os.Stat(oldMongoCLIConfigHome) // check if the dir is already there
+	if err != nil {
+		defaultPermissions := 0700
+		if err = os.Mkdir(mongoCLIConfigHome, os.FileMode(defaultPermissions)); err != nil {
+			return
+		}
```

Comment:
I am not sure why you are checking for the `oldMongoCLIConfigHome` and then create the `mongoCLIConfigHome` if the old dir is not there

Reply:
my bad, this should be `mongoCLIConfigHome`, thank you!